docs/rules/no-default-export.md

```diff
@@ -0,0 +1,63 @@
+# no-default-export
```

Comment:
i think maybe `forbid-default-export` would be a more typical name for a rule like this, and for the opposite of "prefer"

Reply:
ping

Reply:
I was following the naming convention the other rules have - there's also `no-dynamic-require`, `no-named-as-default`, and `no-anonymous-default-export`. So IMHO it wouldn't make sense to use `forbid-${name}` when all the others use `no-`. Also, [ESLint recommends this convention](https://eslint.org/docs/developer-guide/working-with-rules#rule-naming-conventions), so I'm kind of hesitant to deviate from it

Reply:
BTW, I kind of missed these comments somehow; sorry about that. 😦

Reply:
fair enough; this one isn't a blocker or anything.